packages/react/src/components/OverflowSet/OverflowButton.tsx

```diff
@@ -76,8 +76,9 @@ export const OverflowButton = (props: IOverflowSetProps) => {
     [itemSubMenuProvider],
   );
 
-  const newOverflowItems = React.useMemo(() => {
-    let currentOverflowItems: IOverflowSetItemProps[] | undefined = [];
+  const { modifiedOverflowItems, keytipsToRegister } = React.useMemo(() => {
+    const newKeytipsToRegister: IKeytipProps[] = [];
+    let newOverflowItems: IOverflowSetItemProps[] | undefined = [];
```

Comment:
I don't think this needs to be `undefined` since you are explicitly using `!` when setting it to `overflowItems!` below.

Reply:
Resolved by removing the ! at the bottom, probably just leftover from when this was converted to a functional component